lib/pbench/server/auth/__init__.py

```diff
@@ -205,6 +205,8 @@ def wait_for_oidc_server(
         """
         try:
             oidc_server = server_config.get("openid", "server_url")
+            oidc_realm = server_config.get("openid", "realm")
+            cert = server_config.get("openid", "cert_location", fallback=True)
```

Comment:
I wonder if we should have a code comment for that fallback value...since it's "meant" to be a string, but booleans are permitted and handled specially.